qemu/tests/blockdev_full_backup_nonexist_target.py

```diff
@@ -0,0 +1,48 @@
+import re
+
+from virttest.qemu_monitor import QMPCmdError
+
+from provider import backup_utils
+from provider.blockdev_full_backup_base import BlockdevFullBackupBaseTest
+
+
+class BlockdevFullBackupNonexistTargetTest(BlockdevFullBackupBaseTest):
+
+    def prepare_test(self):
+        self.preprocess_data_disks()
+        self.prepare_main_vm()
+        self.prepare_data_disks()
+
+    def do_backup(self):
+        """
+        Backup source image to target image
+        """
+        assert len(
+            self.target_disks) >= len(
+            self.source_disks), "No enough target disks define in cfg!"
+        src_lst = list(map(lambda x: "drive_%s" % x, self.source_disks))
+        dst_lst = list(map(lambda x: "drive_%s" % x, self.target_disks))
```

Comment:
what about src_lst = ['drive_%s' %x for x in self.source_disks] ?

Reply:
Have update, thanks zhenchao.